test/cypress/integration/consumer-tools/find-a-housing-counselor/find-a-housing-counselor.cy.js

```diff
@@ -4,9 +4,14 @@ const page = new FindAHousingCounselor();
 
 describe( 'Find a housing counselor', () => {
   describe( 'Search by ZIP code', () => {
+    beforeEach( () => {
```

Comment:
I think you should document what we're doing here (within the file). Because of the specificity of the fixture, if someone were to come along and say, change the searched zip code, the test would start failing, and it would likely take a while to figure out why.

Reply:
I added some comments in 7f10445. It's not a very extensive comment since none of our tests actually depend on getting a specific response back, just a valid response (i.e. even if you changed the ZIP code the tests would still pass), but hopefully it's enough to point people in the right direction.